packages/application-shell/src/components/application-shell/application-shell.spec.js

```diff
@@ -312,7 +312,7 @@ describe.each`
 
     describe('when user navigates to "/account" route', () => {
       if (renderNodeAsChildren) {
-        it('should trigger a page reload (when served by proxy)', async () => {
+        it.skip('should trigger a page reload (when served by proxy)', async () => {
```

Comment:
Hey @commercetools/shield-team-fe,
I haven't been able to figure out why the console error 'POST http://localhost:8080/graphql net::ERR_FAILED' appears during the execution of this test. Perhaps someone might have any idea